### ISSUE TITLE
How to reduce the Picker size without effecting the UI

### ISSUE THREAD

**Issue by LOG-TAG** [ShawnLin013/NumberPicker#5]
I changed the 
```
        android:layout_width="40dp"
        android:layout_height="100dp"
```

all UI stuffs messed UP, other numbers are over lapping the divider!

**Comment by ShawnLin013** [comment]
You can use the following code to scale NumberPicker
```
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:scaleX="0.7"
        android:scaleY="0.7"
```

**Comment by ShawnLin013** [comment]
Hi @LOG-TAG, just released v1.0.1.
```
<com.shawnlin.numberpicker.NumberPicker
    android:id="@+id/number_picker"
    android:layout_width="wrap_content"
    android:layout_height="wrap_content"
    app:np_width="64"
    app:np_height="180" />
```
